apps/omg_eth/lib/omg_eth/client.ex

```diff
@@ -35,7 +35,7 @@ defmodule OMG.Eth.Client do
     case client.eth_syncing() do
       {:ok, false} -> :ok
       {:ok, _} -> {:error, :geth_still_syncing}
-      {:error, :econnrefused} -> {:error, :geth_not_listening}
+      {:error, _} -> {:error, :geth_not_listening}
```

Comment:
hmmm, how confident are you that you're not hidding what the error "actually" is?

Reply:
yeahh, I think singling out geth errors mostly econnrefused from request errors would be useful. I'll change it

Reply:
a log would to it as well!

Reply:
Ino, how about this?